src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -602,4 +603,9 @@ private function registerSecurityConfiguration(ContainerBuilder $container, XmlF
             $loader->load('security.xml');
         }
     }
+
+    private function registerDefaultsConfiguration(ContainerBuilder $container, array $config): void
+    {
+        $container->setParameter('api_platform.defaults', $config['defaults']);
```

Comment:
Maybe it would better to move it to `registerCommonConfiguration`?

Reply:
As you like.